grow/pods/storage/file_storage.py

```diff
@@ -62,10 +62,9 @@ def write(cls, path, content):
                 pass
             else:
                 raise
-        fp = cls.open(path, mode='w')
-        fp.write(content)
-        fp.close()
-        return fp
+        with cls.open(path, mode='w') as fp:
+            fp.write(content)
+            return fp
```

Comment:
i dunno if any place in the code uses the returned `fp` but doesn't seem to make sense to return anything at all if the `fp` is closed since it would need to be re-opened anyway.

Reply:
I agree, I'll remove it and see if it causes any issues.